.github/workflows/deploy-connect-front.yml

```diff
@@ -43,50 +44,82 @@ jobs:
           path: '**/node_modules'
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
 
-      - name: Build
+      - name: Login in Azure
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Build webapp
         env:
-          # This needs to be true ideally however at the moment the app wont build because of some js errors.
+          # CI needs to be `true` ideally. However at the moment the app wont build because of some js errors.

```

Comment:
```suggestion
          # CI needs to be `true` ideally. However at the moment the app won't build because of some js errors.

```

Reply:
<3 you need to fix this in the code so that we can remove this comment. Hope the comment makes sense, it is mostly written for you :)

Reply:
I think I need some more context - care to elaborate? We can do a quick 2-minute call if you like